VLApi/Controllers/BooksController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Model;
+
+namespace VLApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BooksController : ControllerBase
+    {
+        private readonly ApplicationDbContext _context;
+
+        public BooksController(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Books
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Book>>> GetBook()
+        {
+            return await _context.Book.ToListAsync();
+        }
+
+        // GET: api/Books/5
+        [HttpGet("{id}")]
```

Comment:
Add ProducesResponseType decorations to describe what this controller can return